syncd/Syncd.cpp

```diff
@@ -920,6 +1097,35 @@ sai_status_t Syncd::processBulkOid(
 
     sai_status_t all = SAI_STATUS_SUCCESS;
 
+#ifdef SAI_SUPPORT_BULK_API
+
+    sai_bulk_op_error_mode_t mode = SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR;
+
+    switch (api)
+    {
+        case SAI_COMMON_API_BULK_CREATE:
+            all = processBulkOidCreate(objectType, mode, objectIds, attributes, statuses);
+            break;
+
+        case SAI_COMMON_API_BULK_REMOVE:
+            all = processBulkOidRemove(objectType, mode, objectIds, statuses);
+            break;
+
+        default:
+            all = SAI_STATUS_NOT_SUPPORTED;
+            SWSS_LOG_THROW("api %d is not supported in bulk mode", api);
```

Comment:
you can use %s and sai_serialize_common_api(api).c_str() to get text form of api instead of number